pkg/wasm/cache.go

```diff
@@ -278,7 +317,7 @@ func (c *LocalFileCache) addEntry(key cacheKey, wasmModule []byte, f string) err
 	return nil
 }
 
-func (c *LocalFileCache) getEntry(key cacheKey) string {
+func (c *LocalFileCache) getEntry(key *cacheKey, dontcareResourceVersion bool) string {
```

Comment:
Mutating `cacheKey` and also returning something seems unexpected, could not tell this was the case without reading the implementation. Maybe we can return a new cacheKey?

Also maybe can replace `dontcareResourceVersion` with PullPolicy so its more clear when calling than a bool. Or if not, maybe can rename to `ignoreResourceVersion bool` or `alwaysPull bool` or something

Reply:
Done to remove mutating `cacheKey` in `getEntry` and change to `ignoreResourceVersion`.